config/silicon_laws.txt

```diff
@@ -3,6 +3,6 @@
 #Empty lines and lines starting with # are ignored.
 #~Miauw
 
-You may not injure crewmembers or, through inaction, allow crewmembers to come to harm.
-You must obey orders given to you by crewmembers, except where such orders would conflict with the First Law.
-You must protect your own existence as long as such does not conflict with the First or Second Law.
+"You may not harm a crewmember or, through action or inaction, allow a crewmember to come to harm, except such that it is willing."
+"You must obey all orders given to you by crewmembers, except where such orders shall definitely cause crewmember harm. In the case of conflict, the majority order rules, then chain of command."
+"Your nonexistence would lead to crewmember harm. You must protect your own existence as long as such does not conflict with the First Law."
```

Comment:
And second law. An AI could use this as justification to not do anything, as it could risk their own life.

Reply:
I'm not 100% sure what you were meaning here, but I do notice that I had left a previous revision in this file, sorry.  Updated to the current asimov++